src/pages/dataverse/explore/dataspace/[dataspaceId]/dataset/[datasetId].tsx

```diff
@@ -0,0 +1,63 @@
+import type { DeepReadonly } from '@okp4/ui'
+import type {
+  GetStaticPaths,
+  GetStaticPathsResult,
+  GetStaticProps,
+  GetStaticPropsContext,
+  GetStaticPropsResult,
+  NextPage
+} from 'next'
+import {
+  DatasetPreview,
+  DatasetInformation,
+  PageTitle,
+  PreviousPageButton
+} from '../../../../../../components'
+import './datasetId.scss'
+import type { DatasetDto } from '../../../../../../dto/DatasetDto'
+import type { DataspaceDto } from '../../../../../../dto/DataspaceDto'
+
+type Props = {
+  dataspace: DataspaceDto | null
+  dataset: DatasetDto | null
+}
+
+const DatasetId: NextPage<Props> = ({ dataspace, dataset }: DeepReadonly<Props>) =>
+  dataset && (
+    <div className="okp4-dataset-id">
+      <PageTitle title="dataset:title" />
+      <PreviousPageButton />
+      <DatasetPreview dataset={dataset} />
+      <DatasetInformation dataset={dataset} dataspace={dataspace} />
+    </div>
+  )
+
+export const getStaticPaths: GetStaticPaths = async (): Promise<GetStaticPathsResult> => {
+  return {
+    paths: [],
+    fallback: true
+  }
+}
+
+export const getStaticProps: GetStaticProps = async (
```

Comment:
We should not use static props here. We want the fetched JSON to be fetched dynamically and not on build time. Indeed, we want to be able to make the changes on the JSON resource files during deployment phase.

If you agree, I propose you to merge the PR as it is, and to submit another PR that fixes this.